build/repo.targets

```diff
@@ -5,7 +5,7 @@
     <HtmlJobs>-j "$(RepositoryRoot)src/Benchmarks/benchmarks.html.json"</HtmlJobs>
     <JsonJobs>-j "$(RepositoryRoot)src/Benchmarks/benchmarks.json.json"</JsonJobs>
     <MultiQueryJobs>-j "$(RepositoryRoot)src/Benchmarks/benchmarks.multiquery.json"</MultiQueryJobs>
-    <SignalRJobs>-j $(RepositoryRoot)src/Benchmarks/signalr.json -t SignalR -r signalr --projectFile benchmarks/BenchmarkServer/BenchmarkServer.csproj</SignalRJobs>
+    <SignalRJobs>-j https://raw.githubusercontent.com/aspnet/SignalR/brecon/benchmark/benchmarks/BenchmarkServer/signalr.json -t SignalR -r signalr@brecon/benchmark --projectFile benchmarks/BenchmarkServer/BenchmarkServer.csproj</SignalRJobs>
```

Comment:
`brecon`, is that just for testing? You should merge your PR before merging this one.

Reply:
Yeah, I'll change these values when the signalr change goes in